components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/ConsoleLogger.java

```diff
@@ -26,6 +26,10 @@ public void info(String message, Object... args) {
         out.printf(message, args);
     }
 
+    public void debug(String message, Object... args) {
```

Comment:
Line break missing after the `infoNoNewline` method.

Reply:
✅ Done